plugin.rb

```diff
@@ -9,7 +9,7 @@
 enabled_site_setting :client_performance_enabled
 
 module ::ClientPerformance
-  PLUGIN_NAME = "client-performance"
+  PLUGIN_NAME = "discourse-client-performance"
```

Comment:
Should we be using the `PLUGIN_NAME` value in `SCRIPT_PATH`? 🤔 

Like so

```ruby
  SCRIPT_PATH = "/plugins/#{PLUGIN_NAME}/javascripts/discourse-client-performance.js"
```

And `SCRIPT_HASH` should be using `SCRIPT_PATH` 😅

Reply:
You’re right for the first suggestion but for the second one, there’s a slight difference unfortunately: an extra `public` in the `SCRIPT_HASH`.
I can still use `PLUGIN_NAME` for both though :+1: